### ISSUE TITLE
Wierd pattern in `--exclude` for `lcov` in `make coverage`

### ISSUE THREAD

**Issue by albinahlback** [flintlib/flint2#1269]
I suppose `/usr/*` refers to all other libraries? I usually install my libraries under `$HOME` to avoid needing root privileges. Perhaps exclusions can go into `configure`, and the default is `/usr/*`.

**Comment by fredrik-johansson** [comment]
Yeah, this was just a quick hack to exclude system headers and gmp.h on a typical machine. Feel free to come up with something more systematic. Maybe it's better to just use --include for the files we want.
